tests/integration/test_operator.py

```diff
@@ -1,9 +1,10 @@
 import unittest
 
 from airflow_clickhouse_plugin.operators.clickhouse_operator import ClickHouseOperator
+from tests.integration.util import ClickHouseConnectionEnvVarTestCase
 
 
-class BasicTestCase(unittest.TestCase):
+class BasicTestCase(ClickHouseConnectionEnvVarTestCase):
```

Comment:
Have not this test case been working without this patch? I have not seen any problems in the tests output.